#! /usr/bin/env python
# Task Long configuration

PACKAGE='robot_control_base'
import roslib; roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()
#       Name                    Type            Description
#       Default    Min   Max
gen.add("k_v",           double_t, 0,    "Gain for velocity control",  1.0, 0.0, 10.0)
gen.add("k_alpha",       double_t, 0,    "Gain for angular control",  0.1, 0.0, 10.0)
gen.add("max_velocity",  double_t, 0,    "Max allowed velocity",  0.5, 0.0, 5.0)
gen.add("max_rotational_velocity",  double_t, 0,    "Max allowed rotational velocity",  0.5, 0.0, 6.0)
gen.add("dist_threshold",double_t, 0,    "Distance at which a the target is considered reached",  0.1, 0.0, 10.0)


exit(gen.generate(PACKAGE, "robot_control_base", "GotoXY"))

